crates/aiken-project/src/lib.rs

```diff
@@ -722,7 +535,7 @@ where
                 .map(|match_test| {
                     let mut match_split_dot = match_test.split('.');
 
-                    let match_module = if match_test.contains('.') {
+                    let match_module = if match_test.contains('.') || match_test.contains('/') {
```

Comment:
Orthogonal, but caught (and fixed) that one while testing some other stuff. The filter on modules wasn't working quite right. For example `-m aiken/list` wouldn't actually yield what you intuitively expect.